common/text-level-modifiers/text-level-modifiers-en.html

```diff
@@ -0,0 +1,92 @@
+---
+{
+	"title": "Text-level modifiers",
+	"language": "en",
+	"altLangPage": "text-level-modifiers-fr.html",
+	"dateModified": "2024-11-25",
+	"description": "In the realm of GCweb, text-level modifiers play a pivotal role in shaping the visual and functional aspects of website text. They enable web developers to fine-tune text attributes such as size, color, weight, and style, ensuring that the textual content is not only aesthetically pleasing but also accessible and easy to read."
+}
+---
+
+<div class="wb-prettify all-pre"></div>
+
+<p>In the realm of GCweb, text-level modifiers play a pivotal role in shaping the visual and functional aspects of website text. They enable web developers to fine-tune text attributes such as size, color, weight, and style, ensuring that the textual content is not only aesthetically pleasing but also accessible and easy to read.</p>
+
+<h2><code>.stretched-link</code></h2>
+<p>Used to make an entire containing block clickable, effectively "stretching" a hyperlink over the entire block. Make sure the element that you want to be clickable is positioned. This can be done by adding <code>position: relative</code> to an element.</p>
+<div class="position-relative">
+	<a class="stretched-link" href="#">Stretched link</a>
+</div>
+<pre><code>&lt;div <strong>class=&quot;positon-relative&quot;</strong>&gt;&lt;a <strong>class=&quot;stretched-link&quot;</strong> href="#"&gt;Stretched link&lt;/a&gt;&lt;/div&gt;</code></pre>
+
+<h2><code>.small</code></h2>
+<p>Decreases the font size of the element.</p>
+<p class="small">Example small text</p>
+<pre><code>&lt;p <strong>class=&quot;small&quot;</strong>&gt;...&lt;/p&gt;</code></pre>
+
+<h2><code>.mark</code></h2>
+<dd>Highlights the text.</dd>
+<p class="mark">Example highlighted text</p>
+<pre><code>&lt;p <strong>class=&quot;mark&quot;</strong>&gt;...&lt;/p&gt;</code></pre>
+
+<h2><code>.lead</code></h2>
+<dd>Increases the font size and line height.</dd>
+<p class="lead">
+	Example lead text
+</p>
+<pre><code>&lt;p <strong>class=&quot;lead&quot;</strong>&gt;...&lt;/p&gt;</code></pre>
+
+<h2><code>.no-wrap</code></h2>
+<p>Prevents the text from wrapping.</p>
+<div class="row">
+	<div class="col-xs-6 col-sm-5 col-md-4 col-lg-3">
+		<div class="well">
+			<p class="no-wrap">Example no wrap: xxxxxxxxxxxxxxxxxxxxxxxxxxx</p>
+		</div>
+	</div>
+</div>
+<pre><code>&lt;div class="row">
+	&lt;div class="col-xs-6 col-sm-5 col-md-4 col-lg-3">
+		&lt;div class="well">
+			&lt;p <strong>class="no-wrap"</strong>>Example no wrap: xxxxxxxxxxxxxxxxxxxxxxxxxxx&lt;/p>
+		&lt;/div>
+	&lt;/div>
+&lt;/div></code></pre>
+
+<h2><code>.fnt-nrml</code></h2>
+<p>Resets the <code>font-weight</code> to <code>normal</code>.</p>
+<p>The following element has a normal <strong class="fnt-nrml">font-weight</strong>.</p>
```

Comment:
Because we are undoing the emphasis effect, let's use some character to delimit the emphasis

```suggestion
<p>The following element has a normal <strong class="fnt-nrml">[font-weight]</strong>.</p>
```